tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Models/TestInfo.cs

```diff
@@ -5,7 +5,7 @@ namespace Azure.Sdk.Tools.PerfAutomation.Models
     public class TestInfo
     {
         public string Test { get; set; }
+        public string Class { get; set; }
```

Comment:
This is misleading. How about "fixture"? A common term across numerous languages' common test runners/frameworks.

Reply:
I don't think `Class` is particularly misleading, since a perf test is represented by a class in each language.  But I'm fine with `Fixture` as well.

@AlexanderSher, @g2vinay, @HarshaNalluru, @timovv, @annatisch, @pvaneck, @gearama: Any opinions?

Reply:
`Fixture` sounds good, don't really mind either since this won't flow into the language repos anyway.

Reply:
I chatted with @heaths, he is OK with `Class` since each perf test is represented by a class.  He was thinking each perf test was represented by a method (like unit tests).

Reply:
> `Fixture` sounds good, don't really mind either since this won't flow into the language repos anyway.

This change would flow into the language repos, since it would also rename from `Class` to `Fixture` in all the `perf-tests.yml` files.

Reply:
woah, ok